### ISSUE TITLE
Can not eliminate partition effect in order_cells

### ISSUE THREAD

**Issue by trueth1206** [cole-trapnell-lab/monocle3#275]
Hi, Thank you for creating this great tool. I set the use_partition = FALSE to put all cells in the same pseudotime. Is went well in learn_graph step. But I found the cells in different partitions turned gray after order_cell.

Here is the code I am using:
```{r}
cds <- learn_graph(cds, use_partition = FALSE)
plot_cells(cds, color_cells_by = "cell_type", label_groups_
by_cluster=TRUE, label_leaves=TRUE, label_branch_points=FALSE, group_label_size = 4)
cds  <- order_cells(cds)
plot_cells(cds, color_cells_by = "pseudotime", label_cell_groups=FALSE,  label_leaves=FALSE, label_branch_points=FALSE, graph_label_size=1.5)
```

![14_6_1(25)](https://user-images.githubusercontent.com/49007028/72082193-ac76d000-32f7-11ea-9266-3af3f5e9c8f8.jpg)
![14_6_2](https://user-images.githubusercontent.com/49007028/72082194-ac76d000-32f7-11ea-9320-a13c3a83e9d2.jpg)

Currently is there any way to solve this issue? 

Many thanks.

**Comment by KatjaRM** [comment]
Same issue here. Would be glad for any ideas on how to solve this!

**Comment by brgew** [comment]
Hi,

We made some changes to Monocle3 to try to fix the problems related to running learn_graph(use_partition=FALSE) and my tests suggest that the changes are working. I pushed the changes to the Monocle3 develop repository branch on GitHub: it looks like you can install that branch using the command

devtools::install_github("cole-trapnell-lab/monocle3", ref="develop")

I am eager to hear whether or not this works for others.

**Comment by hpliner** [comment]
Believe this is fixed in 0.2.2 reopen if the issue persists!
